pkg/eventservice/dispatcher_stat.go

```diff
@@ -294,16 +294,18 @@ type wrapEvent struct {
 	msgType int
 	// postSendFunc should be called after the message is sent to message center
 	postSendFunc func()
+	isRedo       bool
```

Comment:
We also don't need add `isRedo` to `WrapEvent`. You can start a pool of workers dedicated to send messages to redo dispatchers(Add a `isRedo` parameter to runSendMessageWorker). `resolvedTsCache` also doesn't need know what is redo.